nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryAPI.java

```diff
@@ -272,6 +272,51 @@ public void testBasics() {
         assertEquals(cal, modified);
     }
 
+    /*
+     * NXP-28727
+     */
+    @Test
+    public void testBasicsCreateDocsWithHierarchy() {
+        DocumentModel root = session.getRootDocument();
+
+        // create the parent doc under the root
+        DocumentModel parent = session.newDocumentModel(new PathRef("/"), "domain", "MyDocType");
+        Calendar parentModifTime = GregorianCalendar.getInstance();
+        parent.setProperty("dublincore", "title", "Title of parent");
```

Comment:
Could you use `DocumentModel#setPropertyValue` and `DocumentModel#getPropertyValue`?

I know the class mix both, but let's add "correct" code :)